src/test/java/com/tngtech/java/junit/dataprovider/internal/convert/ObjectArrayConverterTest.java

```diff
@@ -38,7 +38,7 @@ public void testConvertShouldCreateEmptyVarargsArrayForMissingOnlyVarargsArgumen
         Object[] result = underTest.convert(data, true, parameterTypes);
 
         // Then:
-        assertThat(result).isEqualTo(new Object[] { new int[0] });
+        assertThat(result).containsExactly(new int[0]);
```

Comment:
I more or less did a mechanical replacement. I agree, if the `Object[]` is the focus of these test it's better not to hide that. So either revert or use a helper method with an intention revealing name, maybe something like assertIsObjectArrayWithValues(result, values...).

Of course it up to you, as it's your project. :smile: